starport/cmd/network.go

```diff
@@ -67,6 +70,35 @@ func NewNetwork() *cobra.Command {
 
 var cosmos *cosmosclient.Client
 
+// initializeNetwork initializes event bus, CLIn components such as spinner and returns a new network builder
+func initializeNetwork(cmd *cobra.Command) (
+	*network.Builder,
+	*clispinner.Spinner,
+	func(),
```

Comment:
```suggestion
	cleanup func(),
```

to make it easy to understand we can give a name returned values